src/Application.vala

```diff
@@ -117,6 +117,8 @@ public class Tasks.Application : Gtk.Application {
     public static int main (string[] args) {
         GtkClutter.init (ref args);
         var app = new Application ();
-        return app.run (args);
+        int res = app.run (args);
+        ICal.Object.free_global_objects ();
```

Comment:
Presumably this will not interfere with other apps using ICal?

Reply:
It shouldn't, they're not sharing the memory